src/view.js

```diff
@@ -64,7 +68,8 @@ export class View {
   }
 
   renderStartPage() {
-    if (this.appContainer.hasChildNodes()) this.appContainer.replaceChildren();
+    if (this.#appContainer.hasChildNodes())
+      this.#appContainer.replaceChildren();
```

Comment:
```suggestion
    if (this.#appContainer.hasChildNodes()) {
      this.#appContainer.replaceChildren();
    }
```
There's no error here, but when `if` statements span multiple lines it's more readable to wrap the block of code to be executed in curly brackets `{}`

Reply:
I actually write down all the previous feedback and there was once a suggestion about removing curly brackets for one line arrow function. Ever since I was too obsessed with that and didn't consider the readability of code which is very important too. 😅 Thank you. ❤️